src/assets/data/defaultConfig.ts

```diff
@@ -124,7 +121,7 @@ export const DefaultNotificationRefreshTime: number = 900000 // 15 mins in ms
 
 export const DefaultSourceTypeModel: string = 'aRMT-App'
 export const DefaultSourceTypeRegistrationBody = {
-  sourceTypeCatalogVersion: '1.4.3',
+  sourceTypeCatalogVersion: '1.4.0',
```

Comment:
is it intentional to fetch spec for 1.4.3 and have source-type version of 1.4.0?. Plus why do we roll back on versions?

Reply:
I think this is just for testing now since the schemas and topics for spec 1.4.3 are not yet registered on the server. This will be 1.4.3 when we release the app

Reply:
Yup it was originally 1.4.3 and I had to change it back to 1.4.0 since it's not a source in the management portal yet. I will change it back to 1.4.3.